File: HealthCareClinic/Hospital/Events/Model/Event.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Hospital.Events.Model
{
    public class Event
    {
        public int Id { get; set; }
        public DateTime Timestamp { get; set; }
        public String Content { get; set; }
        public int UserId { get; set; }

        public Event()
        {
        }

        public Event(int id, string content, int userId)
        {
            Id = id;
            Timestamp = DateTime.Now;
            Content = content;
            UserId = userId;
        }

    }
}

File: ManagerApplication/ManagerApp/src/app/model/event.ts
export class Event {
    id: number;
    timestamp: Date;
    content: string;
    userId: number;

    constructor(id: number, timestamp: Date, content: string, userId: number){
        this.id = id;
        this.timestamp = timestamp;
        this.content = content;
        this.userId = userId;
    }
}
